tools/rccl_replayer/README.md

```diff
@@ -0,0 +1,79 @@
+# RCCL REPLAYER
+Collective log replayer tool for RCCL.
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Features](#features)
+3. [How It Works](#how-it-works)
+4. [Installation](#installation)
+5. [Usage](#usage)
+
+## Introduction
+
+Replayer is a dubugging tool designed to analyze and replay collective logs obtained from RCCL (ROCm Communication Collectives Library) runs. It can be a useful tool when trying to recreate problem situations (without as much setup), or as a user-directed utility to run collectives (by crafting their own 'logfile').
+
+## Features
+
+- Parses and validates collective logs from RCCL runs.
+- Detects missing/faulty group calls and provides report.
+- Replays collective calls based on the recorded data.
```

Comment:
We should point out that we do not record data.  We only record which calls were made and use dummy data.